Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,120 @@
+
+### Ceph CSI
+
+Ceph CSI plugins implement an interface between a CSI-enabled Container Orchestrator (CO) and Ceph clusters. For further information, please check out [CSI Driver documentation](https://rook.io/docs/rook/latest/Storage-Configuration/Ceph-CSI/ceph-csi-drivers/).
+
+### Krew Plugin
+
+The [Rook Krew plugin](https://rook.io/docs/rook/latest/Troubleshooting/krew-plugin/) is a tool to help troubleshoot your Rook cluster.
+
+### OpenShift
+
+OpenShift Container Platform is a cloud-based Kubernetes container platform. The foundation of OpenShift Container Platform is based on Kubernetes and therefore shares the same technology.
+
+### Object Bucket Claim (OBC)
+
+An Object Bucket Claim (OBC) is custom resource which requests a bucket (new or existing). For further reference please refer to [OBC Custom Resource](https://rook.io/docs/rook/latest/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-claim/).
+
+### Object Bucket (OB)
+
+An Object Bucket (OB) is a custom resource automatically generated when a bucket is provisioned. It is a global resource, typically not visible to non-admin users, and contains information specific to the bucket.
+
+### Toolbox
+
+The [Rook toolbox](https://rook.io/docs/rook/latest/Troubleshooting/ceph-toolbox/) is a container with common tools used for rook debugging and testing.
+
+## Rook CRDs
+### CephCluster CRDs
+
+Rook allows creation and customization of storage clusters through the custom resource definitions (CRDs). For more information about the different modes of creation of clusters, refer our [documentation](https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/).
+
+### Host Storage Cluster
+
+A [host storage cluster](https://rook.io/docs/rook/latest/CRDs/Cluster/host-cluster/) is one where Rook configures Ceph to store data directly on the host.
+
+### PVC Storage Cluster
+
+In a [PVC-based cluster](https://rook.io/docs/rook/latest/CRDs/Cluster/pvc-cluster/), the Ceph persistent data is stored on volumes requested from a storage class of your choice.
+
+### Stretch Storage Cluster
+
+For environments that only have two failure domains available where data can be replicated, consider the case where one failure domain is down and the data is still fully available in the remaining failure domain. To support this scenario, Ceph has integrated support for [stretch clusters](https://rook.io/docs/rook/latest/CRDs/Cluster/stretch-cluster/).
+
+### External Storage Cluster
+
+An [external cluster](https://rook.io/docs/rook/latest/CRDs/Cluster/external-cluster/) is a Ceph configuration that is managed outside of the local K8s cluster.
+
+### CephBlockPool CRD
+
+Rook allows creation and customization of storage pools through the custom resource definitions (CRDs). The following settings are available for pools. For more information and examples refer this [documentation](https://rook.io/docs/rook/latest/CRDs/Block-Storage/ceph-block-pool-crd/).
```

Comment:
CephBlockPool is the CRD used by Rook to allow creation and customization of storage pools. For more information and examples, please refer to [the documentation](https://rook.io/docs/rook/latest/CRDs/Block-Storage/ceph-block-pool-crd/).

Reply:
Let's restructure the "Rook allows ...." definitions to look like this